tests/e2e/nvidia-container-toolkit_test.go

```diff
@@ -166,4 +168,51 @@ var _ = Describe("docker", Ordered, func() {
 			Expect(referenceOutput).To(Equal(out4))
 		})
 	})
+
+	Describe("CUDA Forward compatibility", Ordered, func() {
```

Comment:
Nice! I notice the test cases get skipped since our holodeck installs the latest driver: https://github.com/NVIDIA/nvidia-container-toolkit/actions/runs/13331020349/job/37235261007?pr=906#step:10:95

Any plans to downgrade the version of the driver we install in our e2e tests?

Reply:
There are plans, but I have not been able to persuade holodeck to install a lower driver version yet. I need to figure out which driver versions are available on the instance as a starting point.